backend/wmg/data/snapshot.py

```diff
@@ -114,11 +114,23 @@ def _update_latest_snapshot_identifier() -> Optional[str]:
         return None
 
 
+def _build_data_path_prefix():
+    wmg_config = WmgConfig()
+    rdev_prefix = os.environ.get("REMOTE_DEV_PREFIX")
+    if rdev_prefix:
+        return rdev_prefix.strip("/")
+    elif "data_path_prefix" in wmg_config.config:
+        return wmg_config.data_path_prefix
+    else:
+        return ""
```

Comment:
I'm wondering if this config-related complexity should be encapsulated in WmgConfig. We have a [precedent](https://github.com/chanzuckerberg/single-cell-data-portal/blob/1eca356857b5f7c1994cdfd1977523069cd26ed5/backend/corpora/common/corpora_config.py#L34) of doing that.

Reply:
On 2nd thought, we probably need a better way overall to think about all the rdev-specific config hacks. This is well encapsulated here as well, since it's only for snapshot.

Reply:
I'm happy to move this to `WmgConfig` if you prefer. I agree it's probably better.